tests/Tutor.Infrastructure.Tests/Unit/EventStore/EventStoreManager.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using Tutor.Infrastructure.Database.EventStore;
+using Tutor.Infrastructure.Database.EventStore.Postgres;
+using Tutor.Infrastructure.Tests.TestData.EventStore;
+
+namespace Tutor.Infrastructure.Tests.Unit.EventStore
+{
+    public class EventStoreManager : IDisposable
+    {
+        private readonly string _connectionString = $"Server=localhost;Port=5432;Database=event-store-test;User ID=postgres;Password=super;Integrated Security=false;Pooling=true;";
+        private EventContext _eventContext;
+
+        public IEventStore EventStore { get; private set; }
+
+        public EventStoreManager()
```

Comment:
Wouldn't this make the test an "integration" test?

Reply:
I was pretty unsure about how to categorize it, it might make more sense to call it an integration test.

Reply:
Moved to integration in cc5665d9268535bb310118fb13485b7cf847701a.